### PR TITLE
Add remove

### PR DESCRIPTION
**Pull request by marilirulita**
add functions add and remove elements
update list array in local store
add edit elements function

### CODE REVIEW

File: src/addElement.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+/* eslint-disable import/no-cycle */
```
Comment (MrBrN197) [inline src/addElement.js]: - [ ] Please avoid using `eslint-disable` to get around ESlint errors. ESlint errors are usually denoting an underlying problem with the way your code is setup up. please remove this ESLint error and address any issues that you may have. I know it's much easier to just disable these errors but it's much better to learn to write clean code.  
Your issues are coming from the way you are importing separate files.

File: src/deleteElements.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+/* eslint-disable import/no-cycle */
```
Comment (MrBrN197) [inline src/deleteElements.js]: - [ ] Please fix this ESlint issue as mentioned in my comment above. https://github.com/marilirulita/ToDoList/pull/2#discussion_r783492012

File: src/display.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
+/* eslint-disable import/no-cycle */
```
Comment (MrBrN197) [inline src/display.js]: - [ ] Please fix this ESlint issue as mentioned in my comment above. https://github.com/marilirulita/ToDoList/pull/2#discussion_r783492012

File: src/editTask.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
+/* eslint-disable import/no-cycle */
```
Comment (MrBrN197) [inline src/editTask.js]: - [ ] Please fix this ESlint issue as mentioned in my comment above. https://github.com/marilirulita/ToDoList/pull/2#discussion_r783492012

**Review by MrBrN197 — CHANGES_REQUESTED** [review]
## Hi, @marilirulita  👋 

You've done a really nice job so far.
There are some issues that you still need to work on to go to the next project but you are almost there!
## To highlight 👏👏
- ✔ javascript is functioning well

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Cathella — APPROVED** [review]
## 🟩  Status: Approved 🎉💪🏻

### Hi @marilirulita 👋🏻, 

You did a tremendous job implementing the requested changes 💪🏻.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**